src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
The interface name `Area` seems to be a noun, which is good, but it could be more descriptive of the action it represents. Consider renaming it to something like `AreaCalculable` or `AreaProvider` to better convey the intended use. This aligns with the checklist item: Don't use verbs for class/interface names.